README.md

```diff
@@ -12,44 +12,48 @@ git clone https://github.com/irods/irods_client_http_python.git
 ## Usage
 To use the wrapper, follow the steps listed below.
 
-Import the package.
-```
+```.py
 from irods_client import IrodsClient
-```
 
-Create an instance of the wrapper with the base url of the iRODS server to be accessed. \<host>, \<port>, and \<version> are placeholders, and need to be replaced by appropriate values.
-```
+# Create an instance of the wrapper with the base url of the iRODS server to be accessed.
```

Comment:
Let's break a few characters early so the rendered code snippet doesn't require a horizontal scrollbar.

_This applies to all lines within this code snippet._